languages/clojure/exercises/concept/international-calling-connoisseur/.docs/instructions.md

```diff
@@ -0,0 +1,77 @@
+In this exercise you'll be writing code to keep track of international dialing codes via an international dialing code hashmap.
+
+The map uses an integer for its keys (the dialing code) and a string (country name) for its values.
+
+You have 11 tasks which involve dialing-codes.
+
+### 1. Create a pre-populated hashmap
+
+There exists a pre-populated hashmap which contains the following 3 dialing codes: "United States of America" which has a code of 1, "Brazil" which has a code of 55 and "India" which has a code of 91. Define the `countries` var to store the pre-populated hashmap:
+
+```clojure
+countries
+;;=> {1 "United States of America", 55 "Brazil", 91 "India"}
+```
+
+### 2. Add a country to a map
+
+Add "United Kingdom" with a dialing code of 44 to the map created in task 1:
+
+```clojure
+(add-country countries 44 "United Kingdom")
+;;=> {1 "United States of America", 44 "United Kingdom", 55 "Brazil", 91 "India"}
+```
+
+### 3. Get the country name matching a country code
+
+Check that a country with the country name for dialing code 55
```

Comment:
It looks like there is some missing text here

Reply:
Ah yes that's confusing... it's the same as the C# one I forked it from.